### PR TITLE
feat: Convert frontend-elements submodule to a components library[DEV-1760]

### PR DESCRIPTION
**Pull request by benyam7**
- We have managed to make `frontend-elements` an npm package and decreased size of package greatly.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import alias from "@rollup/plugin-alias";
+// import peerDepsExternal from "rollup-plugin-peer-deps-external";
```
Comment (jay-dee7) [inline rollup.config.js]: @benyam7 let's remove this

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+      file: pkg.module,
+      format: "esm",
+    },
+  ],
+  plugins: [
+    alias({
+      entries: [{ find: "@assets", replacement: "./assets/images" }],
+    }),
+    // peerDepsExternal(),
```
Comment (jay-dee7) [inline rollup.config.js]: and this

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
Amazing work @benyam7 let's remove the comments and we're good with this PR

File: rollup.config.js (hunk 1 above)
Comment (benyam7) [reply rollup.config.js]: nice catch! 👍🏾

File: rollup.config.js (hunk 2 above)
Comment (benyam7) [reply rollup.config.js]: nice catch! 👍🏾

**Comment by benyam7** [comment]
> Amazing work @benyam7 let's remove the comments and we're good with this PR

Thanks man!

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**